src/utils/db.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('dotenv').config();
```

Comment:
It's better to load environment variables in the entry point of your application. Loading them in 'db.js' might cause issues if other modules depend on environment variables before this file is imported. 🌍